FlowCrypt/Functionality/Services/Contacts Service/Models/RecipientWithSortedPubKeys.swift

```diff
@@ -26,21 +26,12 @@ struct RecipientWithSortedPubKeys {
 extension RecipientWithSortedPubKeys {
     init(_ recipient: Recipient, keyDetails: [KeyDetails] = []) {
         self.email = recipient.email
-        self.name = recipient.name
+        self.name = recipient.name ?? keyDetails.first?.users.first ?? email
```

Comment:
That's a good idea, use name from pubkey when otherwise missing. But I'm not sure what the `?? email` will do in practice, I hope it will not send an email message with a recipient like `email@example.com <email@example.com>`. And I also hope that `email@example.com <email@example.com>` will not be submitted to the FES upload API. Can you please verify that?

Reply:
Yes - in that case, should send without name, as it was before.